crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -159,10 +159,82 @@ impl Device {
     /// can be confirmed as the creator and owner of the `m.room_key`.
     pub fn is_owner_of_session(&self, session: &InboundGroupSession) -> bool {
```

Comment:
I wonder if this should have a name that reflects the fact we are looking for cryptographic authentication here. `is_authenticated_owner_of_session`, maybe.

Reply:
There is no other type of ownership, but I don't feel strongly about this. So if you prefer this name and nobody objects I'm fine with changing the name as well.